### PR TITLE
Implement the design of sneak peak wireframe

### PR DESCRIPTION
**Pull request by altontonn**
# Blog app - Views 💎 
In this PR:-
1️⃣ I Implemented the design from the sneak peek wireframes given.

- I used CSS to style my app

2️⃣  I added some data to your database
3️⃣ I used some of the methods used when processing models

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+          <td>
+            <h2><%= user.name %><br>
+            </h2>
+            Number of posts: <%= user.posts_counter %>
+          </td>
+        </tr>
+      <% end %>
+    </table>
+  </body>
+</html>
```
Comment (ShadyShawkat) [inline app/views/users/index.html.erb]: - [ ] As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Note%3A%20Analyze%20the%20wireframes%20deeply%20and%20try%20to%20identify%20all%20the%20necessary%20elements%20and%20how%20a%20user%20would%20navigate%20from%20one%20page%20to%20another) requirement. Please make sure that the user can navigate between the pages to access all of the specified routes. For example, in the `Users#index` view, the user can click on one of the users' cards to access the `Users#show` view of the clicked card.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+        </td>
+      </tr>
+    </table>
+    <div class="bio">
+      <h3>Bio</h3>
+      <p><%= @user.bio %></p>
+    </div>
+    <% @posts.each do |post| %>
+      <div class="bio bio2">
+        <div class="box1">
```
Comment (ShadyShawkat) [inline app/views/users/show.html.erb]: - [ ] Please make sure to use the methods that you have created in the models (showing only the three most recent posts), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,47 @@
...
+              <h3>Post #<%= post.id %></h3>
+              <p><%= post.text %>
+              </div>
+              <div class="box2">
+                <p>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+              </div>
+            </div>
+            <div class="bio">
+              <% post.comments.each do |comment| %>
+                <b><%= @user.name %></b>: <%= comment.text %>
```
Comment (ShadyShawkat) [inline app/views/posts/index.html.erb]: - [ ] Please make sure to use the methods that you have created in the models (showing only the five most recent comments), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @altontonn, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @altontonn 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**